hugolib/page.go

```diff
@@ -362,7 +363,16 @@ func (p *Page) ReadFrom(buf io.Reader) (int64, error) {
 }
 
 func (p *Page) analyzePage() {
-	p.WordCount = len(p.PlainWords())
+	p.WordCount = 0
+	for _, word := range p.PlainWords() {
+		runeCount := utf8.RuneCountInString(word)
+		if len(word) == runeCount {
+			p.WordCount++	
+		} else {
+			p.WordCount += runeCount
```

Comment:
This doesn't look like an universal truth. This may hold water for Chinese, Japanese etc., but will give wrong result for other UTF-8 rich languages. Also, I added the `RuneCount`method on Page for this purpose some time ago.